packages/forms-web-app/src/dynamic-forms/has-questionnaire/journey.js

```diff
@@ -30,7 +30,7 @@ class HasJourney extends Journey {
 		);
 
 		this.sections.push(
-			new Section('Constraints, designations and other issues', 'constraints')
+			new Section('Constraints, designations and other issues', 'constraints', this.urlMap)
```

Comment:
Or if we want to take this even further we could also add a function inside  the journey class like so:

`
constructor() {
...
this.sections.push(
			this.createSection('Constraints, designations and other issues', 'constraints')
				.addQuestion(questions.appealTypeAppropriate)
}
createSection(name, segment) {
		return new Section(name, segment, this.urlMap);
	}
`

Feel free to ignore this unless you find it very useful :)